packages/concerto-cto/lib/parser.pegjs

```diff
@@ -1711,22 +1715,44 @@ ImportType
   }
 
 ImportTypes
-    = ImportToken __ ns:QualifiedNamespaceDeclaration ".{" _ types:commaSeparatedIdentifiers _ "}" __ u:FromUri? {
+    = ImportToken __ ns:QualifiedNamespaceDeclaration ".{" _ types:commaSeparatedTypes _ "}" __ u:FromUri? {
+    	const { aliasedTypes, remainingTypes } = types.reduce((acc, type) => {
+        if (type.$class === "concerto.metamodel@1.0.0.AliasType") {
+          acc.aliasedTypes.push(type);
+          acc.remainingTypes.push(type.name);
+        } else {
+          acc.remainingTypes.push(type);
+        }
+        return acc;
+      }, { aliasedTypes: [], remainingTypes: [] });
     	const result = {
             $class: "concerto.metamodel@1.0.0.ImportTypes",
             namespace: ns,
-            types,
+            types:remainingTypes,
+            ... aliasedTypes.length >0 && {aliasedTypes},
+            
         };
         u && (result.uri = u);
         return result;
     }
 
-commaSeparatedIdentifiers
-  = head:$Identifier _ tail:(',' _ @$Identifier _ )*
-  {
-    return [head, ...tail];
-  }
+AliasedIdentifier 
+    = name:$Identifier _ $AsToken _ aliasName:$Identifier{
+      const result = {
+        "$class":"concerto.metamodel@1.0.0.AliasType",
+        name:name,
+        aliasName:aliasName
+      };
+      return result;
+    }
+  
+Types
```

Comment:
Let's make this rule name more specific. `Types` is a very ambiguous term for us! :-)

Reply:
How about `IdentifierTypeExpression` or simply `TypeExpression`